lib/ui/window/platform_configuration.cc

```diff
@@ -67,8 +69,14 @@ void PlatformConfiguration::DidCreateIsolate() {
                   Dart_GetField(library, tonic::ToDart("_drawFrame")));
   report_timings_.Set(tonic::DartState::Current(),
                       Dart_GetField(library, tonic::ToDart("_reportTimings")));
-  windows_.insert(std::make_pair(
-      0, std::make_unique<Window>(0, ViewportMetrics{1.0, 0.0, 0.0, -1})));
+
+  // TODO(loicsharma): This should only be created if the embedder enables the
+  // implicit view.
+  // See: https://github.com/flutter/flutter/issues/120306
+  windows_.insert(
+      std::make_pair(kImplicitViewId,
+                     std::make_unique<Window>(
+                         kImplicitViewId, ViewportMetrics{1.0, 0.0, 0.0, -1})));
```

Comment:
You should be able to use map::emplace() here, I'd think.

Reply:
Good idea, that's much better!